app/data/jsonSchemaForm/projectManagerSchema.ts

```diff
@@ -0,0 +1,16 @@
+const projectManagerSchema = {
+  $schema: "http://json-schema.org/draft-07/schema",
+  type: "object",
+  title: "Project Manager",
+  required: ["cifUserId"],
+  properties: {
+    cifUserId: {
+      type: "number",
+      title: "Project Manager",
+      default: undefined,
+      anyOf: undefined,
```

Comment:
This is where we need to include `anyOf`, otherwise rjsf's [deepEqual](https://github.com/rjsf-team/react-jsonschema-form/blob/8335ad1966c7e249225d9dce50766750785d8e77/packages/core/src/components/Form.js#L38) implementation trips and triggers a change event on form load - which in turn triggers an update mutation.
Ajv doesn't care when validating on the server side.